### PR TITLE
Show example button is consistent in all pages

### PR DESCRIPTION
**Pull request by itolimaesther**
This text can be deleted when you are ready to make the pull request: 

For each PR, please add the following, if appropriate: 

- a brief description of what you have done
In other to solve the issue of inconsistent "Example button", I moved the code for the "See example" button from views.cljs file to the bottom right by placing it in results.cljs (before the heading Results)
and I also moved the variable declaration of (ex) to be able to generate the example code.

- what issues it addresses, if any
It addresses "Show me an example" buttons are inconsistent between pages #465

- what needs to be done in order to test the PR
fork and clone the repro the enter the directory
Install Leiningen  if you don't have installed already  => sudo apt install leiningen
npm install
lein dev

### CODE REVIEW

**Comment by heralden** [comment]
I feel the placement is still inconsistent. How about placing the *Show Example* button on the region search page, to be directly left of the *Search* button? This should make it pretty similar to the upload page. This would mean that the *Search* button would no longer be full-width.

**Review by heralden — CHANGES_REQUESTED** [review]
Forgot to submit my review. Please see my comment above!

**Comment by itolimaesther** [comment]
Alright @uosl

**Comment by itolimaesther** [comment]
show (1)

@uosl I have corrected it now.

File: less/site.less [hunk 1]
```diff
@@ -416,26 +416,7 @@ button.btn, button.cta {
...
-    position: fixed;
-    top: 0;
-    left: 0;
-    width: 2em;
-    height: 2em;
-    padding: 0;
-    border: none;
-    outline: none;
-    box-shadow: none;
+.btn-default-2{
```
Comment (heralden) [inline less/site.less]: Looks like something went wrong here during merging? The `p.success`, `p.failure` and `textarea.hidden-clipboard` selectors are needed, so can you add them back?

I also suggest we make the CSS more maintainable by moving the `.btn-default-2` selector into the `less/components/regions.less`, nested appropriately. You should also rename it to `.example-button` so it's more obvious that this styling applies specifically to that one button.

**Review by heralden — CHANGES_REQUESTED** [review]
This is much better :rocket: 

There are some changes to the CSS that needs to be done before I merge this. Hope you can check them out.

File: less/site.less (hunk 1 above)
Comment (itolimaesther) [reply less/site.less]: Alright

**Comment by itolimaesther** [comment]
@uosl I have made the changes

**Review by heralden — APPROVED** [review]
This looks great. Thank you for your help! :100:

**Pull request merged**